### PR TITLE
Annie/updating bp team directory 2 18

### PR DESCRIPTION
**Pull request by aanxniee**
### Implementation description

- Changed term to 1236 and updated team/role/picture link for existing members
- Created new object for new members

### Steps to test

1.

### What should reviewers focus on?

-

### Checklist

- [ ] My PR name is descriptive and in imperative tense
- [ ] I have run the linter
- [ ] I have requested a review from other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: constants/members.json [hunk 1]
```diff
@@ -464,15 +472,15 @@
     },
     {
       "name": "William Tran",
-      "role": "developer",
-      "term": 1235,
-      "teams": ["focus-on-nature"],
+      "role": "project lead",
+      "term": 1236,
+      "teams": ["ma"],
```
Comment (Jess2772) [inline constants/members.json:477]: assuming you mean "mp" here haha, once we confirm the team name + id pairs you can update

File: constants/members.json [hunk 2]
```diff
@@ -3152,29 +3160,29 @@
...
-      "role": "vp talent",
-      "term": 1235,
+      "role": "vp scoping",
+      "term": 1236,
       "teams": ["executive"],
       "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FVPT_Tracy_Wai.jpeg?alt=media&token=f8bdd126-24b6-4594-80ca-7f4440933725&_gl=1*4frx90*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMTg1OC4wLjAuMA.."
     },
     {
       "name": "Edward Ho",
       "role": "vp scoping",
```
Comment (Jess2772) [inline constants/members.json:3177]: edward (eddie) ho is actually pm for abtc now, so youll need to update this

Comment (aanxniee) [reply constants/members.json:3177]: main spreadsheet is wrong? @heet

**Review by HeetShah — CHANGES_REQUESTED** [review]
just need to update Edward --> Eddie Ho and to PM for `abtc`. Looks good otherwise!

**Review by HeetShah — APPROVED** [review]
LGTM

File: constants/members.json [hunk 3]
```diff
@@ -1714,9 +1722,9 @@
     },
     {
       "name": "Soha Khan",
-      "role": "product manager",
-      "term": 1231,
-      "teams": ["ccbc"],
+      "role": "director lead",
```
Comment (LucyWuu) [inline constants/members.json:1725]: @HeetShah is this a new role that's added this term?

**Pull request closed without merging**